protos/core.proto

```diff
@@ -303,6 +309,12 @@ message Assertion {
   reserved 4, 5, 6;
 }
 
+enum ActionHermeticity {
+  UNKNOWN = 0;
```

Comment:
Just so I understand - this is because with proto3 there is no way to tell the difference between set and default value?

Reply:
I don't think you actually need to do all this for the purpose of validation. you could just validate this when setting the config, using hasOwnProperty (validation happens before we proto-ify these values)

Reply:
Yep